### DISCUSSION TITLE
Paperless NGX [ERROR] in line 93: during update

### DISCUSSION THREAD

**Discussion by Flosen86** [tteck/Proxmox#4018] [Help Needed] [answered]
Hi all,

gettings this message

[ERROR] in line 93: exit code 0: while executing command pip install -r requirements.txt &> /dev/null

Update is run using the script command

bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/paperless-ngx.sh)"

running it manually, creates that error.

root@lxcPaperless:/opt/paperless# python3 -m pip install --upgrade pip
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
python3-xyz, where xyz is the package you are trying to
install.

If you wish to install a non-Debian-packaged Python package,
create a virtual environment using python3 -m venv path/to/venv.
Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
sure you have python3-full installed.

If you wish to install a non-Debian packaged Python application,
it may be easiest to use pipx install xyz, which will manage a
virtual environment for you. Make sure you have pipx installed.

See /usr/share/doc/python3.11/README.venv for more information.
note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.

[notice] A new release of pip is available: 24.1.2 -> 24.3.1
[notice] To update, run: python3 -m pip install --upgrade pip

**Comment by Mellowlynx** [comment, marked as answer]
#3300
